src/cljs/pyregence/components/forecast_tabs.cljs

```diff
@@ -22,6 +25,14 @@
               hover-text
               :top
               [:label {:style    ($forecast-label (= current-forecast key))
-                       :on-click #(select-forecast! key)}
+                       :on-click
+                       (fn []
+                         (when (= key :active-fire)
+                           (when (and
+                                  (zero? @!/active-fire-count)
+                                  (zero? @!/active-fire-tab-click-count))
+                             (toast-message! "No active fires."))
+                           (swap! !/active-fire-tab-click-count inc))
+                         (select-forecast! key))}
```

Comment:
What happens in `select-forecast!` if there is no active fires really?
Wouldn't be better to avoid calling `select-forecast!` function in such a case?